rust-runtime/aws-smithy-http-server/examples/BENCHMARKS.md

```diff
@@ -0,0 +1,84 @@
+# Smithy Rust Server SDK benchmarks
+
+This Pokémon Service has been benchmarked on different type of EC2 instances
+using [wrk](https://github.com/wg/wrk).
+
+<!-- vim-markdown-toc Marked -->
+
+* [2022-03-04](#2022-03-04)
+    * [c6i.8xlarge](#c6i.8xlarge)
+        * [Full result](#full-result)
+    * [c6g.8xlarge](#c6g.8xlarge)
+        * [Full result](#full-result)
+
+<!-- vim-markdown-toc -->
+
+## 2022-03-04
+
+The benchmark run against the `empty_operation()` operation, which is just
+returning an empty output and can be used to stress test the framework overhead.
+
+### c6i.8xlarge
+
+* 32 cores Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz
+* 64 Gb memory
+* Benchmark:
+    - Duration: 10 minutes
+    - Connections: 1024
+    * Threads: 16
+* Result:
+    - Request/sec: 1_608_742
+    * RSS memory: 72200 bytes
```

Comment:
```suggestion
    - RSS memory: 72200 bytes
```

Should we link or spell out https://en.wikipedia.org/wiki/Resident_set_size ? I didn't know this term.

Reply:
Could also be a footnote https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/

Reply:
Yeah, why not..